website/docs/docs/deploy/webhooks.md

```diff
@@ -19,6 +19,7 @@ dbt Cloud retries sending each event five times. dbt Cloud keeps a log of each w
 
 ## Prerequisites
 - You have a dbt Cloud account that is on the [Team or Enterprise plan](https://www.getdbt.com/pricing/). 
+- You have set up a multi-tenant deployment environment for your dbt jobs. For more information, refer to [Tenancy](/docs/cloud/about-cloud/tenancy). 
```

Comment:
You have a multi-tenant deployment in dbt Cloud. For more information, refer to [Tenancy](/docs/cloud/about-cloud/tenancy).